jaeger/charts/linkerd-jaeger/templates/NOTES.txt

```diff
@@ -0,0 +1,15 @@
+The Linkerd Jaeger extension was successfully installed 🎉
+
+To help you manage your Linkerd service mesh you can install the Linkerd CLI by running:
+
+  curl -sL https://run.linkerd.io/install | sh
+
+Alternatively, you can download the CLI directly via the Linkerd releases page:
```

Comment:
Should we remove the CLi installation docs and replace this with cmds like `check`, etc

Reply:
I think we can leave the CLI install instructions, and additionally mention `linkerd check` to make sure everything is ok, etc.